### PR TITLE
CI: Add "shiping" to CodeSpell whitelist

### PR DESCRIPTION
**Pull request by Pandapip1**
See https://github.com/ethereum/EIPs/pull/5521

### CODE REVIEW

**Comment by Pandapip1** [comment]
(It's the name of somebody)

File: .codespell-whitelist [hunk 1]
```diff
@@ -12,3 +12,4 @@ acn
 ende
 sting
 complies
+shiping
```
Comment (xinbenlv) [inline .codespell-whitelist:15]: I am unsure why `shiping` to whitelist....

**Comment by abcoathup** [comment]
Shiping Chen is an author on EIP5521.
Is there a way to skip the author section rather than having to allow list everywhere?

**Comment by Pandapip1** [comment]
> Is there a way to skip the author section rather than having to allow list everywhere?

Unfortunately not. I wish this was the case though.

**Comment by xinbenlv** [comment]
Without alternatives, I am in favor of checking in this commit to allow list Shiping's name

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**